input/pagecontent/smart-web-messaging.md

```diff
@@ -455,6 +467,59 @@ targetWindow.postMessage({
 }, targetOrigin);
 ```
 
+If an app were to request the entire contents of the scratchpad by submitting a
+`scratchpad.read` without specifying a `location` parameter, the EHR might respond with
+a bundle containing multiple resources.
+
+Assuming the scratchpad contained an object like this:
+
+```json
+{
+  "ServiceRequest/1": {
+    "resourceType": "ServiceRequest",
+    "status": "draft"
+  },
+  "MedicationRequest/1": {
+    "resourceType": "MedicationRequest",
+    "status": "draft"
+  }
+}
+```
+
+The resulting response from the EHR might look like this:
+
+```js
+{
+  "responseToMessageId": "<the id of the request message>",
+  "messageId": "<some new uid>",
+  "payload": {
+    "scratchpad": {
+      "resourceType": "Bundle",
+      "id": "scratchpad-read-all-example",
+      "type": "searchset",
+      "total": 2,
+      "status": "200 OK",
+      "entry": [
+        {
+          "resource": {
+            "resourceType": "ServiceRequest",
+            "id": 1,
```

Comment:
This `1` needs quotes; in general, it'd be good to put this whole FHIR bundle through a validator make sure there aren't other issues lurking.

Reply:
https://fhirvalidator.org/ is one option that Mark announced recently on Zulip.

Reply:
I've updated the example to be a valid r5 collection bundle.  However, I'm not keen on the medication reference that I've used.  I did try to use a non-reference, but I got hung up on the validator (which was in flux at the time).

Is there a simple example I can crib from?

Reply:
Let's stick to R4 (R5 doesn't exist yet!)

Reply:
https://www.hl7.org/fhir/us/core/MedicationRequest-self-tylenol.json.html has an example of a medication using `medicationCodeableConcept`.